mods/ra/maps/soviet-05/main.lua

```diff
@@ -0,0 +1,212 @@
+		return actor.Type == "fact" and actor.Owner == GoodGuy
+	end)
+
+	return #ConYard >= 1
+end
+
+CheckForSPen = function()
+	SPens = Map.ActorsInBox(Map.TopLeft, Map.BottomRight, function(actor)
+		return actor.Type == "spen"
+	end)
+
+	return #SPens >=1
+end
+
+RunInitialActivities = function()
+	if Map.Difficulty == "Hard" then
+		Expand()
+		ExpansionCheck = true
+	else
+		ExpansionCheck = false
+	end
+
+	Trigger.AfterDelay(1, function(harvester)
+		Harvester.FindResources()
+		Helper.Destroy()
+	end)
+	
+	Reinforcements.Reinforce(player, sov1, sov1Path, 0, function(soldier)
+		soldier.AttackMove(SovietBasePoint.Location)
+	end)
+
+	Actor.Create("camera", true, { Owner = player, Location = GreeceBasePoint.Location })
+	Actor.Create("camera", true, { Owner = player, Location = SovietBasePoint.Location })
+
+	startmcv.Move(MCVStartMovePoint.Location)
+	Runner1.Move(RunnerRetreatPoint.Location)
+	Runner2.Move(RunnerRetreatPoint.Location)
+	Runner3.Move(RunnerRetreatPoint.Location)
+
+	ProduceInfantry()
+	Trigger.AfterDelay(DateTime.Minutes(2), ProduceShips)
+
+	Media.PlaySpeechNotification(player, "ReinforcementsArrived")
+
+	if Map.Difficulty == "Hard" or Map.Difficulty == "Medium" then
+		Trigger.AfterDelay(DateTime.Seconds(25), Gguy1)
+	end
+	Trigger.AfterDelay(DateTime.Minutes(2), Gguy1)
+	Trigger.AfterDelay(DateTime.Minutes(5), Gguy1)
+end
+
+Expand = function()
+	if not ExpansionCheck then
+		if not mcvtransport.IsDead then
+		mcvtransport.Move(mcvGGLoadPoint.Location)
+		end
+		if not mcvGG.IsDead then
+		mcvGG.Move(mcvGGLoadPoint.Location)
+		end
+		Reinforcements.Reinforce(GoodGuy, {"dd", "dd"}, {ReinfNorthPoint.Location, NearDockPoint.Location}, 0, function(ddsquad)
+			ddsquad.AttackMove(NearExpansionPoint.Location)
+		end)
+		ExpansionCheck = true
+		Trigger.ClearAll(mcvGG)
+		Trigger.ClearAll(mcvtransport)
+		Trigger.AfterDelay(DateTime.Seconds(3), function(load)
+			if not mcvtransport.IsDead then
+			mcvGG.IsInWorld = false
+			else
+				return
+			end
+			if not mcvtransport.IsDead then
+				mcvtransport.LoadPassenger(mcvGG)
+				mcvtransport.Move(mcvGGUnloadPoint.Location)
+			else
+				return
+			end
+			Trigger.AfterDelay(DateTime.Seconds(10), function(unload)
+				if not mcvtransport.IsDead then
+					mcvtransport.UnloadPassengers()
+				end
+				Trigger.AfterDelay(DateTime.Seconds(2), function(move)
+					if not mcvGG.IsDead then
+					mcvGG.Move(MCVDeploy.Location)
+					end
+					Trigger.AfterDelay(DateTime.Seconds(4), function(deploy)
+						if not mcvGG.IsDead then
+						mcvGG.Deploy()
+						end
+						if not mcvtransport.IsDead then
+							mcvtransport.Move(ReinfNorthPoint.Location)
+						end
+						IslandTroops()
+						Trigger.AfterDelay(DateTime.Seconds(7), function(leave)
+							if not mcvtransport.IsDead then
+								mcvtransport.Destroy()
+							end
+							BuildBase()
+						end)
+					end)
+				end)
+			end)
+		end)
+	else
```

Comment:
Redundant else. What would be better is if you had 
`if (ExpansionCheck) return` at the top. This will remove a level of indentation too :)